packages/core-mobile/app/screens/tokenManagement/TokenManagementItem.tsx

```diff
@@ -20,6 +20,7 @@ const TokenManagementItem: FC<Props> = ({ id, name, image, symbol }) => {
 
   const isBlacklisted = useSelector(selectIsTokenBlacklisted(id))
 
+  // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
we shouldn't be ignoring this.  get co-pilot and it will tell you which return type to use here.

Reply:
done!